gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/nfp_node_driver.py

```diff
@@ -707,9 +748,65 @@ def _is_node_order_in_spec_supported(self, context):
             raise InvalidNodeOrderInChain(
                     node_order=allowed_chain_combinations)
 
+    def _get_consumers_for_provider(self, context, provider):
```

Comment:
What is the advantage of getting these details here Vs in orchestrator? Is this only to avoid multiple REST calls for other data fields in "nfp_create_nf_data" variable? and to keep all the information retrieval of variable in one place?